install.sh

```diff
@@ -270,6 +293,36 @@ libsecp256k1_install()
     popd
 }
 
+clightning_build ()
+{
+    ./configure \
+        --enable-developer \
+        --enable-experimental-features \
+        --prefix="${jm_root}"
+    $make
```

Comment:
suggest using:
`$make -j1`
to avoid running out of memory while building: https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000#issuecomment-962425193

Reply:
Alternative - take `num_cores()` approach from #1061.

Reply:
This should be solved in #1177.